### ISSUE TITLE
provide a configuration option to set the MathJax-node options

### ISSUE THREAD

**Issue by jinzhen-lin** [lierdakil/mathjax-pandoc-filter#5]
In some cases, I must set some MathJax-node options to make this filter work properly.

For example, I may want to set `extensions` of MathJax by 

```javascript
mjAPI.config({
  extensions: ["xxxxx"]
});
```

I may also want to set `cjkCharWidth` to make the Chinese characters in the math expressions display properly.

```javascript
mjAPI.typeset({
  cjkCharWidth: 15,
  math: mn,
  format: inline ? 'inline-TeX' : 'TeX',
  svg: true,
  speakText: false,
  linebreaks: false,
}
```

So can you provide a configuration option to set the MathJax-node option? For example, allow user to provide a JSON file to set the MathJax-node option.

**Comment by lierdakil** [comment]
The easiest way to do this for me would probably be to just read objects
from YAML metadata, and pass those straight to te configuration functions.
Say, 'mathjax' to configure and 'mathjax.typeset' to typeset. Does that
sound reasonable?

чт, 20 дек. 2018 г., 15:28 Jinzhen Lin <notifications@github.com>:

**Comment by jinzhen-lin** [comment]
that sounds like a great idea!

**Comment by lierdakil** [comment]
Published as mathjax-pandoc-filter@0.4.0. TBH, I didn't test it all that much, but it seems to work. Feel free to open new issues if you find something amiss.
